src/test/end-to-end/helpers/PuppeteerHelper.js

```diff
@@ -13,18 +13,29 @@ class PuppeteerHelper extends Helper {
 
     async waitForNavigationToComplete(locator) {
         const page = this.helpers[helperName].page;
+        const promises = [];
 
-        const promises = [
-            page.waitForNavigation({timeout: 60000, waitUntil: ['domcontentloaded', 'networkidle0']}) // The promise resolves after navigation has finished
-        ];
```

Comment:
This is interesting - unsure as to why this wouldn't work for xui, both are webpages loaded in a web browser? May cause complications down the line e.g. newCase